### PR TITLE
Update test debugging

### PR DESCRIPTION
**Pull request by johnsonw**
Update test debugging

---
This change is [Reviewable](https://reviewable.io/reviews/whamcloud/integrated-manager-for-lustre/2065)

### CODE REVIEW

File: iml-system-test-utils/src/lib.rs [hunk 1]
```diff
@@ -273,14 +273,22 @@ impl Config {
     pub fn get_setup_config(&self) -> String {
         match &self.test_type {
             TestType::Rpm => format!(
-                r#"USE_STRATAGEM = {}
-BRANDING = "{}""#,
+                r#"USE_STRATAGEM={}
+BRANDING={}
+LOG_LEVEL=10
```
Comment (nlinker) [inline iml-system-test-utils/src/lib.rs:278]: This makes me thinking what the 10 is. I was expecting `LOG_LEVEL` to be usual one of `trace`, `debug`, `warn` etc.
How is it used?

Comment (ip1981) [reply iml-system-test-utils/src/lib.rs:278]:
```
$ python
>>> import logging
>>> logging.DEBUG
10
```

File: iml-system-test-utils/src/lib.rs [hunk 2]
```diff
@@ -273,14 +273,22 @@ impl Config {
...
+RUST_LOG=debug
```
Comment (ip1981) [inline iml-system-test-utils/src/lib.rs:279]: Just a note, this makes less noise:

```
RUST_LOG=debug,hyper=warn,lapin=warn,warp=info
```

File: iml-system-test-utils/src/lib.rs (hunk 1 above)
Comment (johnsonw) [reply iml-system-test-utils/src/lib.rs:278]: `LOG_LEVEL` is the python's log level and it's an integer. Here is the python repl output:
```
>>> import logging
>>> logging.DEBUG
10
>>> logging.INFO
20
```

File: iml-system-test-utils/src/lib.rs (hunk 2 above)
Comment (johnsonw) [reply iml-system-test-utils/src/lib.rs:279]: If we find that we need to change this i'm willing to do so in a separate patch.

Comment (jgrund) [reply iml-system-test-utils/src/lib.rs:279]: I think we will actually want the verbosity in this case.

**Pull request merged**